### ISSUE TITLE
Choose a URL to load into the preview pane

### ISSUE THREAD

**Issue by briandant** [akamai/launchpad#4]
@dshafik: Is it right that you want the preview pane to open to the course login page? https://launchpad.akamaideveloper.com/login. If so, unfortunately it cannot be framed:
![Screenshot 2019-05-03 13 26 30](https://user-images.githubusercontent.com/1884902/57163831-1e7d3980-6da7-11e9-9cf1-689b318f5b7d.png)

Do you have another page you'd like us to load?

**Comment by dshafik** [comment]
Nope, we want to open it to the _current_ page the user is viewing (e.g. `window.location`). I can solve the framing issues at the Edge if need be.

**Comment by briandant** [comment]
Okay, I'd say we should disable this for now. I'd rather not try to force this in for your demo on Monday.
